src/System.Net.Http/src/System/Net/Http/HttpClientHandler.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Net.Http
+{
+    public partial class HttpClientHandler : HttpMessageHandler
+    {
+        // This partial implementation contains members common to all HttpClientHandler implementations.
+        
+        private static bool UseManagedHandler => Environment.GetEnvironmentVariable("COMPlus_UseManagedHttpClientHandler") == "true";
```

Comment:
It could be anything... I was just matching all of the env vars in coreclr. I expect this along with lots of other things will change before something ships. My care factor is super low on this; if you have a strong preference for something else right now, just share it and I'll change it.